pkg/config/config.go

```diff
@@ -694,8 +694,10 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("cluster_agent.server.read_timeout_seconds", 2)
 	config.BindEnvAndSetDefault("cluster_agent.server.write_timeout_seconds", 2)
 	config.BindEnvAndSetDefault("cluster_agent.server.idle_timeout_seconds", 60)
+	config.BindEnvAndSetDefault("cluster_agent.refresh_on_cache_miss", true)
 	config.BindEnvAndSetDefault("cluster_agent.serve_nozzle_data", false)
-	config.BindEnvAndSetDefault("cluster_agent.advanced_tagging", false)
```

Comment:
We usually don't remove config settings unless it's a hidden config and/or absolutely necessary to prevent forward incompatibilities for users. It's not a blocker - just bringing this up so that the implications of the change are fully considered.

Reply:
Good point! However, in our case (in PCF), we don't expose agent settings directly to the users, they are abstracted away by other [properties](https://github.com/DataDog/datadog-agent-boshrelease/blob/master/jobs/dd-agent/spec) at the BOSH release level.